Installation/Action/klant.extend.action.json

```diff
@@ -0,0 +1,15 @@
+{
+    "title": "ExtendObjectAction",
+    "$id": "https://commongateway.nl/action/klant.extend.action.json",
+    "$schema": "https://docs.commongateway.nl/schemas/Action.schema.json",
+    "version": "0.0.1",
+    "listens": ["extend.object"],
```

Comment:
Where / when do you throw this event?

Reply:
As stated in the PR description: it is not enabled by default due to the open questions